client/src/cmdlfem4x50.c

```diff
@@ -779,6 +781,12 @@ static int CmdEM4x50Reader(const char *Cmd) {
         // iceman,  misuse of return status code.
         int now = resp.status;
 
+        // prevent massive stack corruption if unexpected results from device.
+        if (now > EM4X50_NO_WORDS) {
+            PrintAndLogEx(WARNING, "word count was: %d, limiting to %d", now, EM4X50_NO_WORDS);
+            now = EM4X50_NO_WORDS;
+        }
+
         if (now > 0) {
 
```

Comment:
this one can be "   if (now) {

Reply:
This PR is focused on CodeQL changes.  The existing code was as shown, and the change is stylistic in nature.  Thus, it is outside the scope of this PR.